src/install.coffee

```diff
@@ -318,7 +346,10 @@ class Install extends Command
     for name, version of @getPackageDependencies()
       do (name, version) =>
         commands.push (next) =>
-          @installRegisteredPackage({name, version}, options, next)
+          if version.startsWith('file:.')
```

Comment:
You could also parse these with Node's [url module](https://nodejs.org/dist/latest-v8.x/docs/api/url.html) and dispatch on `protocol`:

<details>

```
$ node
> const url = require('url')
undefined
> url.parse('file:./foo/bar/baz')
Url {
  protocol: 'file:',
  slashes: null,
  auth: null,
  host: null,
  port: null,
  hostname: null,
  hash: null,
  search: null,
  query: null,
  pathname: './foo/bar/baz',
  path: './foo/bar/baz',
  href: 'file://./foo/bar/baz' }
> url.parse('merge-conflicts')
Url {
  protocol: null,
  slashes: null,
  auth: null,
  host: null,
  port: null,
  hostname: null,
  hash: null,
  search: null,
  query: null,
  pathname: 'merge-conflicts',
  path: 'merge-conflicts',
  href: 'merge-conflicts' }
>
```
</details>

It might be a little more flexible and easier to extend in the future. This should be fine for now, though 👍

Reply:
No problem, I'll add that!

Reply:
On second thought, I think it might be better if we start parsing the version string at an earlier point in the code so that we can attach more metadata to it related to whether it's a version string, url path, etc.  I think I'll defer that for now so that I can get this change merged and get the Atom PR moving forward.